src/components/results/securityanalysis/security-analysis-result-utils.ts

```diff
@@ -57,37 +64,29 @@ export const flattenNmKResultsContingencies = (
             subjectLimitViolations = [],
             contingency,
         }: ConstraintsFromContingencyItem) => {
-            if (!!subjectLimitViolations.length) {
-                const {
-                    contingencyId,
-                    computationStatus,
-                    elements = [],
-                } = contingency || {};
-                rows.push({
-                    contingencyId,
-                    contingencyEquipmentsIds: elements.map(
-                        (element) => element.id
-                    ),
-                    computationStatus,
-                    violationCount: subjectLimitViolations.length,
-                });
-                subjectLimitViolations?.forEach((constraint: Constraint) => {
-                    const { limitViolation = {} as LimitViolation, subjectId } =
-                        constraint || {};
+            const { contingencyId, status, elements = [] } = contingency || {};
+            rows.push({
+                contingencyId,
+                contingencyEquipmentsIds: elements.map((element) => element.id),
+                status,
+                violationCount: subjectLimitViolations.length,
+            });
+            subjectLimitViolations?.forEach((constraint: Constraint) => {
+                const { limitViolation = {} as LimitViolation, subjectId } =
+                    constraint || {};
 
-                    rows.push({
-                        subjectId,
-                        limitType: intl.formatMessage({
-                            id: limitViolation.limitType,
-                        }),
-                        limit: limitViolation.limit,
-                        value: limitViolation.value,
-                        loading: limitViolation.loading,
-                        side: limitViolation.side,
-                        linkedElementId: contingencyId,
-                    });
+                rows.push({
+                    subjectId,
+                    limitType: intl.formatMessage({
+                        id: limitViolation.limitType,
+                    }),
+                    limit: limitViolation.limit,
+                    value: limitViolation.value,
+                    loading: limitViolation.loading,
+                    side: limitViolation.side,
+                    linkedElementId: contingencyId,
```

Comment:
You have to add acceptableDuration and limitName to display name and duration values